src/taskpane/components/SearchField.tsx

```diff
@@ -0,0 +1,17 @@
+import { ISearchBoxStyles, SearchBox } from "@fluentui/react";
+import React from "react";
+
+interface SearchFieldProps {
+  onFilterChange: any;
```

Comment:
Specify the correct type `(event?: React.ChangeEvent<HTMLInputElement>, newValue?: string) => void`?